introduction.md

```diff
@@ -1,2 +1,8 @@
 This file introduces ti8m-hic to the deadly vipers dojo.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?